lomond/constants.py

```diff
@@ -11,10 +13,12 @@
 # https://tools.ietf.org/html/rfc6455
 WS_VERSION = 13
 
-# Only report major.minor in version
-_version_identifier = '.'.join(__version__.split('.')[:2])
-
 # User agent sent with websocket request
-# Default will be DataplicityLomond/<major>.<minor
-# e.g "DataplicityLomond/0.1"
-USER_AGENT = 'DataplicityLomond/{}'.format(_version_identifier)
+USER_AGENT = 'Lomond/{} {}/{} {}/{} ({})'.format(
```

Comment:
I added an `agent` parameter to the WebSocket object. I'd assume people would do something like:

```
WebSocket('wss://example.org', agent="MyClient/0.1.0 " + constants.USER_AGENT)
```

To be honest, I think a `client` parameter may be too much hand-holding.